composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "sonata-project/formatter-bundle",
+    "name": "sonata-project/formatter-bundle-tworks",
```

Comment:
?

Reply:
Please change this - I had to change the name to use it for internal testing instead of the one from package list.